jobs/geo/write_building_footprints.py

```diff
@@ -51,21 +55,30 @@ def write_building_footprints(conn):
         s3 = s3fs.S3FileSystem(anon=False)
         s3.put(
             f"{file_prefix}.parquet",
-            "s3://dof-demographics-dev-us-west-2-public/parquet/",
+            f"s3://dof-demographics-dev-us-west-2-public/{kind}_building_footprints/parquet/{file_prefix}.parquet",
```

Comment:
Here is a proposed change to the naming structure in S3. Now that there are two datasets, it felt weird to me to have `parquet/` or `shp/` be the top-level folders. It now looks like
```
global_ml_building_footprints/parquet/county_fips_001.parquet
global_ml_building_footprints/shp/county_fips_001.zip
us_building_footprints/parquet/county_fips_001.parquet
us_building_footprints/shp/county_fips_001.zip
```

Reply:
To clarify, the top-level folders will now be either US-based or Global-based with both zip and parquet files stored in either?

Reply:
Correct. It's in there now if you want to take a look